src/main/java/zpi/entity/User.java

```diff
@@ -0,0 +1,66 @@
+package zpi.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.List;
+
+@Entity
+@Data
+@NoArgsConstructor
+@Table(name = "users")
+public class User implements UserDetails {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    private String firstName;
+    private String lastName;
+    private String login;
+    private String password;
+    @Enumerated(value = EnumType.STRING)
```

Comment:
Tak samo jak w PR ze scheme - musimy obgadać czy chcemy tu mieć many to many czy user może mieć tylko jedna rolę

Reply:
Zamienilem wszystko na podejscie z 1 rolą. Z uwagi na rozmiar naszej aplikacji i planowane role uważam to za lepsze podejście